Library/Formula/pandoc.rb

```diff
@@ -24,7 +24,12 @@ class Pandoc < Formula
 
   def install
     cabal_sandbox do
-      cabal_install "--only-dependencies"
+      args = []
+      if MacOS.version <= :lion
+        # disable AESNI for OSX <= 10.7
+        args << "--constraint='cryptonite -support_aesni'"
```

Comment:
Try removing the quotes. These arguments aren't being passed to a shell, so shell splitting/quoting rules don't apply.